set1/Q4.cpp

```diff
@@ -0,0 +1,47 @@
+//https://practice.geeksforgeeks.org/problems/edit-distance/0
+#include<iostream>
+#include<algorithm>
+using namespace std;
+char str1[101], str2[101];
+int dp[101][101];
```

Comment:
Current Algo is fine. But space can be reduced even more. You can mention that in comments.